src/index.html

```diff
@@ -6,8 +6,245 @@
     <title>Miami</title>
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
+    <link
+      rel="icon"
+      href="images/Icon-Corona-virus.svg"
```

Comment:
```suggestion
      href="images/Icon-Crown.svg"
```
This code will be seen by recruiters. Not all of them will appreciate such humour)) Better to change